apps/payments/next/app/[locale]/[offeringId]/checkout/[interval]/[cartId]/start/page.tsx

```diff
@@ -122,7 +121,7 @@ export default async function Checkout({ params }: { params: CheckoutParams }) {
 
       {!session ? (
         <h2
-          className="font-semibold text-grey-600 text-lg mt-14 mb-5"
+          className="font-semibold text-grey-600 text-lg mt-10 mb-5"
```

Comment:
Clarification question. Why change it to `mt-10`? SP2 has it as `mt-14` at the moment.

Reply:
SGTM!